editor/gui/editor_title_bar.cpp

```diff
@@ -80,6 +80,47 @@ void EditorTitleBar::gui_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+void EditorTitleBar::_notification(int p_what) {
+	if (p_what == NOTIFICATION_SORT_CHILDREN) {
+		Control *prev = nullptr;
+		Control *base = nullptr;
+		Control *next = nullptr;
+		for (int i = 0; i < get_child_count(); i++) {
+			Control *c = as_sortable_control(get_child(i));
+			if (!c) {
+				continue;
+			}
+			if (base) {
+				next = c;
+				break;
+			}
+			if (!c->has_meta("_editor_tab_bar")) {
```

Comment:
Looks like the title bars have only one centered control. You could make it a property in EditorTitleBar (something like `set_center_control(main_editor_button_hb)` etc.)

Reply:
Done,